4_visualize.R

```diff
@@ -0,0 +1,44 @@
+source('4_visualize/src/plot_results.R')
+
+p4 <- list(
+  # Plots for each lake-gcm combo
+  tar_target(
+    p4_20yr_gcm_preds_ice,
+    plot_20yr_gcm_preds_ice(p2_glm_uncalibrated_run_groups, 
```

Comment:
Is this the only plot that you did not include an example of in your PR comment?

Reply:
Yup  - only because it's nearly identical to the  `p4_20yr_average_preds_ice` plots, but for a single GCM:
![Site_20yr_preds_ice_nhdhr_105569520_GFDL](https://user-images.githubusercontent.com/54007288/160720186-671b800f-8476-42a5-af49-38a7f6be0b07.png)